kiwmi/luak/kiwmi_server.c

```diff
@@ -287,6 +317,8 @@ static const luaL_Reg kiwmi_server_methods[] = {
     {"spawn", l_kiwmi_server_spawn},
     {"stop_interactive", l_kiwmi_server_stop_interactive},
     {"unfocus", l_kiwmi_server_unfocus},
+    {"verbose", l_kiwmi_server_verbose},
```

Comment:
I think `verbose` is somewhat of an odd name.

Got any better ideas (I'm so far very happy with avoiding any `set_` type thingies)?

Reply:
I was thinking about this earlier, but couldn’t find anything. Maybe that’s because `verbosity` is a *terminus technicus* for me and I don’t know its actual (original) meaning.

`verbosely` seems weird too. `set_verbosity` is `set_*`. `change_verbosity` is just `set_*` called differently.

Maybe `log_with_{level|verbosity}`, but that’s long.

Reply:
Overloading it depending on the argument (`kiwmi:verbosity()` to get and `kiwmi:verbosity(4)` to set) would work, but is rather ugly.

Reply:
So if there’s no good name apart from `set_verbosity`, overloading is your preference?

Reply:
I talked to some other people and they can't really come up with anything fancy either.

Overloading is really disgusting, I think set_verbosity is the way.